app/tests/v2/base_v2.py

```diff
@@ -83,30 +82,25 @@ def setUp(self):
             "body" : "this is a test body_3"
         }
 
-        self.register = self.client.post(
-            '/api/v1/users/register',
-            data=json.dumps(self.user_1),
-            content_type='application/json')
-
-        self.login = self.client.post(
-            '/api/v1/users/login',
-            data=json.dumps(self.user_1),
-            content_type='application/json')
+        # self.login = self.client.post(
+        #     '/api/v2/auth/login',
+        #     data=json.dumps(self.user_1),
+        #     content_type='application/json')
 
-        self.create_meetup = self.client.post(
-            '/api/v1/meetups',
-            data=json.dumps(self.meetup_1),
-            content_type='application/json')
+        # self.create_meetup = self.client.post(
```

Comment:
remove code that is commented out before pushing it

Reply:
Thanks will correct that in my next PR